File: Software Technologies/Exam SoftUni Lab 10.Jan.2019/CSharp/SoftUniLabs/SoftUniLabs/Models/Lab.cs
using System.ComponentModel.DataAnnotations;

namespace SoftUniLabs.Models
{
    public class Lab
    {      
        //TODO

        [Key]
        public int Id { get; set; }

        public string Name { get; set; }

        public int Capacity { get; set; }

        public string Status { get; set; }
    }
}

File: Software Technologies/Exam SoftUni Lab 10.Jan.2019/JavaScript/softunilabs/models/lab.js
const Sequelize = require('sequelize');

module.exports = function (sequelize) {
    //TODO: Implement me

    let Lab = sequelize.define('Lab', {
        name:{type: Sequelize.TEXT,required: true, allowNull: false,
        },
        capacity:{
            type: Sequelize.INTEGER,required: true, allowNull: false,
        },
        status:{
            type: Sequelize.TEXT,required: true, allowNull: false,
        }
    }, {
        timestamps:false
    });
    return Lab;
};